modules/rhel-compute-updating.adoc

```diff
@@ -61,6 +62,13 @@ By default, the base OS RHEL with "Minimal" installation option enables firewall
 As of {product-title} 4.11, the Ansible playbooks are provided only for {op-system-base} 8.  If a {op-system-base} 7 system was used as a host for the {product-title} 4.10 Ansible playbooks, you must either upgrade the Ansible host to {op-system-base} 8, or create a new Ansible host on a {op-system-base} 8 system and copy over the inventories from the old Ansible host.
 ====
 
+.. On the machine that you run the Ansible playbooks, update the Ansible package:
```

Comment:
I know this is beyond the scope of this PR, but moving forward please be mindful that if a step in a procedure is to run a command, our guidelines specify that the step include some form "run/enter the following command". There are several patterns that can be followed [1].

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output